lib/xml_builder.ex

```diff
@@ -219,7 +219,11 @@ defmodule XmlBuilder do
 
   defp format(:xml_decl, 0, options) do
     encoding = Keyword.get(options, :encoding, "UTF-8")
-    ~s|<?xml version="1.0" encoding="#{encoding}"?>|
+    standalone = case Keyword.get(options, :standalone, "") do
```

Comment:
Default should be `false` instead of empty string `""`, since `standalone` it's a boolean

Reply:
Yes, makes sense and done…